### ISSUE TITLE
docker compose AWS LogGroup retain after stack deletion

### ISSUE THREAD

**Issue by thorfi** [docker/compose#8959]
**Description**

`docker compose down` always deletes the LogGroup - it would be nice to have the ability to use an external LogGroup, e.g. by specifying `x-aws-loggroup-external: yes` or similar option.

**Steps to reproduce the issue:**

1. Create a docker-compose.yml file and deploy in AWS as usual
2. docker compose down
3. docker compose up

**Describe the results you received:**

Associated LogGroup is deleted after step 2, and re-created after step 3.

**Describe the results you expected:**

If option is given to retain logs, LogGroup should be created if it is missing "outside" the stack and then specified as a target instead of being created.

**Comment by ndeloof** [comment]
ECS support is implemented by https://github.com/docker/compose-cli
please report on the relevant repository
